readme.md

```diff
@@ -111,7 +140,14 @@ Useful if you need multiple config files for your app or module. For example, di
 Type: `string`<br>
 Default: The `name` field in the package.json closest to where `conf` is imported.
 
-You only need to specify this if you don't have a package.json file in your project.
+You only need to specify this if you don't have a package.json file in your project or if there isn't a name defined within it.
```

Comment:
```suggestion
You only need to specify this if you don't have a package.json file in your project or if it doesn't have a `"name"` field.
```

Reply:
Done, also replaced it in other 3 places.